src/constants/media.ts

```diff
@@ -3,12 +3,13 @@
 export const IMAGE = 'image'
 export const AUDIO = 'audio'
 export const VIDEO = 'video'
+export const MODEL_3D = 'model'
```

Comment:
```suggestion
export const MODEL_3D = 'model_3d'
```

This should avoid the conflict with PR #1203 and fulfil the requirement set in issue #1119 about the key name for the meta search sources.

Reply:
PR #1204 targeting this branch contains this correction.